src/javaRosa.js

```diff
@@ -1976,6 +1976,7 @@ define([
             );
 
             ret = ret.concat([
+                'appearance',
```

Comment:
Curious why this is added here and not in `getAdvancedProperties` in core.js? Appearance seems unrelated to JavaRosa (itext).